### PR TITLE
cronjob for fixing missing outcome codes

### PR DESCRIPTION
**Pull request by psweeting1**
## What does this pull request do?

We need to rerun a job which fixes cases which are not being billed correctly.

This also needs to be scheduled so it happens regularly before the 4th of each month.

## Checklist

[LGA-2680](https://dsdmoj.atlassian.net/browse/LGA-2680)

[LGA-2680]: https://dsdmoj.atlassian.net/browse/LGA-2680?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by Pheonnexx — DISMISSED** [review]
LGTM

File: helm_deploy/cla-backend/templates/cron.yaml [hunk 1]
```diff
@@ -77,4 +77,24 @@ spec:
...
\ No newline at end of file
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-fix-missing-outcome-codes
+  labels:
+    {{- include "cla-backend.labels" . | nindent 4 }}
+spec:
+  schedule: "0 5 4 * *"
```
Comment (BenMillar-CGI) [inline helm_deploy/cla-backend/templates/cron.yaml:88]: Just a query on this, please dismiss this I am mistaken. 

However, this CronJob is going to run on the 4th of each month at 5AM, however, the ticket says "The task has been scheduled to run before the 4th of each month.".

Is it okay if the task runs before the start of the work day, or could this be an issue? Is there a reason why this shouldn't be run on the 3rd of each month at 23:00, for example?

Comment (psweeting1) [reply helm_deploy/cla-backend/templates/cron.yaml:88]: A good question. 4th and after is when they run their big reports. By doing on the 4th in the morning, they'll have a fresh (and anything was missing would be filled) database with no missing codes. This leave little space for anyone to add more cases that could have blanks in them.

**Review by BenMillar-CGI — DISMISSED** [review]
Looks good to me! 👍

**Review by BenMillar-CGI — APPROVED** [review]
LGTM! 👍

**Review by Pheonnexx — APPROVED** [review]
LGTM 🚢 🎈

**Review by BenMillar-CGI — APPROVED** [review]
LGTM 👍 
YAML Formatting is the worst.

**Pull request merged**